.ci/kata-find-stale-skips.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Description: Look for tests that are currently being skipped, but which
+#   should no longer be (since the issue they are being skipped on is now
+#   closed).
+
+set -e
+
+info()
+{
+	local -r msg="$*"
+
+	echo "INFO: $msg"
+}
+
+# This is a crude heuristic and may break at any time. However, using the
+# github API is equally problematic - unauthenticated access is rate-limited
+# and authenticated access is difficult (2FA for example).
+is_github_issue_closed()
+{
+	local -r issue="$1"
+
+	# Look for the magic github token denoting the issue is still open
+	curl -sL "$issue"|grep -q 'div class="State State--green"' && echo "no" && return
```

Comment:
what about converting `$issue` to an API url, then curl and grep for `"state": "open"`?
```
issue=$(echo $issue | sed -e 's|github.com|api.github.com/repos|')

```

For the authentication, we can use a @katabuilder token so we do not have problems when retrieving the information

Reply:
I like it! Users still might get caught by the github rate-limit, but it's atleast now more reliable when it *does* run :)

Branch updated.